lib/rust/profiler-macros/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "profiler-macros"
+version = "0.1.0"
+edition = "2021"
+
```

Comment:
1. Please add the `authors` section just as in other Cargo.toml files.
2. Our convention here is to prefix the names of the packages with `enso-`, so this package would be named `enso-profiler-macros`. We are using this convention with other libraries too, like `prelude`, `optics`, `logger`, etc. The idea is that this way we have a way smaller chance of crashing with a name of a library registered on crates.io and until we publish these libraries as separate crates, we keep their names prefixed.